lib/components/viewers/stop-viewer.js

```diff
@@ -293,12 +307,17 @@ class StopViewer extends Component {
 const mapStateToProps = (state, ownProps) => {
   const showUserSettings = getShowUserSettings(state)
   const stopViewerConfig = getStopViewerConfig(state)
+  const stopLookup = state.otp.transitIndex.stops
+  const stopData = stopLookup[state.otp.ui.viewedStop.stopId]
+  const nearbyStops = stopData?.nearbyStops?.map(stopId => stopLookup[stopId])
   return {
     autoRefreshStopTimes: state.otp.user.autoRefreshStopTimes,
     favoriteStops: state.otp.user.favoriteStops,
     homeTimezone: state.otp.config.homeTimezone,
+    nearbyStops,
+    showNearbyStops: state.otp.config.nearbyRadius > 0,
```

Comment:
Should probably be:

```suggestion
    showNearbyStops: state.otp.config.stopViewer?.nearbyRadius > 0,
```

Reply:
Added in https://github.com/opentripplanner/otp-react-redux/pull/423/commits/b9eb8c201221a4a676c54c34e58f3bc50a093945.